src/components/InputHeader/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addTodo } from '../../actions';
+
+function InputHeader() {
+  const [text, setText] = useState('');
+
+  const dispatch = useDispatch();
+
+  const handleChange = (e) => setText(e.target.value);
+
+  const capitalizeFirstLetter = (string) => (
+    string.charAt(0).toUpperCase() + string.slice(1)
+  );
+  const handleEnterKey = (e) => {
```

Comment:
what is `e` ?

Reply:
It's "event"

Reply:
ok, please declare it clarify follow `Clean Code` book and push new commit